data.sql

```diff
@@ -23,3 +23,48 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg,
 VALUES ('Blossom', '1998-10-13', 3, 't', 17.0, '');
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
 VALUES ('Ditto', '2022-05-14', 4, 't', 22.0, '');
```

Comment:
- [ ] In milestone 2, you hardcoded the data entry for the `species` column. Kindly remove it because it would cause an error. Remember that `species` has its table and it is connected to the `animals` table through a **_foreign key_**. Therefore, there is no need to retain the `species` column in the `animals` table. Kindly remove the hardcoded data from milestone 2 👍 💻

Reply:
@Gambit142 my friend it is required in milstone 2. I created species table in milstone 3. If I didn't write it in milestone 2, my pr not approved :) But I will remove it ;)

Reply:
@otmanTR, I understand that you need it to run queries in milestone 2 but you need to remove them in milestone 3 because you have a `species` table and a **_foreign key_** in the `animals` table that references the `species` table. I hope it is clearer now. 😄